### DISCUSSION TITLE
Differences Between Two Pause Methods in react-native-video

### DISCUSSION THREAD

**Discussion by gt-zhangyaochen** [TheWidlarzGroup/react-native-video#4025] [Q&A]
### What are you trying to do?

I noticed that `react-native-video` provides two different ways to pause video playback. Could someone explain the differences between these two methods and why both are offered? Specifically, I'm curious about the implications for maintenance and whether having multiple ways to achieve the same functionality could lead to inconsistencies or difficulties in managing the codebase.

**Comment by freeboub** [comment]
Both are the same thing. Code base is quiet the same, no big difficulty to maintain it for now. 
Historically the package was made to map api from Video of html5 that's why pause is a property.
We recently add the function api for performance improvement . Using the function api will avoid some component rerendering when it is not really mandatory.

**Reply by gt-zhangyaochen** [reply]
thx bro
